pydantic/_internal/_model_construction.py

```diff
@@ -367,8 +367,8 @@ def inspect_namespace(  # noqa C901
                 )
             else:
                 raise PydanticUserError(
-                    f"A non-annotated attribute was detected: `{var_name} = {value!r}`. All model fields require a "
-                    f"type annotation; if `{var_name}` is not meant to be a field, you may be able to resolve this "
+                    f'A non-annotated attribute was detected: `{var_name} = {value!r}`. All model fields require a '
+                    f'type annotation; if `{var_name}` is not meant to be a field, you may be able to resolve this '
                     f"error by annotating it as a `ClassVar` or updating `model_config['ignored_types']`.",
                     code='model-field-missing-annotation',
```

Comment:
This change also seems odd to me, as it's not changing the last line. I wonder what happens when we use:
```
[tool.ruff.flake8-quotes]
multiline-quotes = 'double'
```
In the `pyproject.toml`

Reply:
Actually, it should be `'double'` by default, so I don't think that'll change anything. This modification still strikes me as odd though.

Reply:
Yea good eye there. Let me try the change out and see what happens.

Reply:
Btw, your diligence is impressive and I appreciate it.

Reply:
It looks like
```
[tool.ruff.flake8-quotes]
multiline-quotes = 'double'
```

is already implemented on line 179:

```
[tool.ruff]
line-length = 120
extend-select = ['Q', 'RUF100', 'C90', 'UP', 'I', 'D', 'T']
extend-ignore = ['D105', 'D107', 'D205', 'D415']
flake8-quotes = {inline-quotes = 'single', multiline-quotes = 'double'}
mccabe = { max-complexity = 14 }
isort = { known-first-party = ['pydantic', 'tests'] }
target-version = "py37"
extend-exclude = ['pydantic/v1', 'tests/mypy/outputs']
```

Reply:
If I attempt to add the first code block it naturally raise a conflict error.

Reply:
Good catch. I'll leave this as unresolved for now given that I'm still curious about the change the `ruff` formatter made, but I don't think it's a problem in terms of the `flake8-quotes` configuration.

Reply:
This strikes me as a bug -- we should be using double quotes for multi-line strings regardless of the formatter quote setting. Will look into it shortly and report back.

Reply:
The last line doesn't get changed because changing the quote style would require escaping the `'`. See my other comment below.

Reply:
Sorry, as usual, Micha is right -- this is an implicit concatenation, not a multiline string. So it's trying to use single quotes for each segment, and then skipping those that _contain_ a single quote.

Reply:
Ah, I see. This happened a few times with implicit concatenations, and I thought the pattern was odd at first glance bc it always seemed to be skipping the last line, but that seems to always be where we have a single quote based on the nature of many of our strings. Thanks!